src/main/java/org/opentripplanner/netex/mapping/StationMapper.java

```diff
@@ -120,15 +120,22 @@ private WgsCoordinate mapCoordinate(StopPlace stopPlace) {
         stopPlace.getId() + " " + stopPlace.getName()
       );
       List<WgsCoordinate> coordinates = new ArrayList<>();
-      for (Object it : stopPlace.getQuays().getQuayRefOrQuay()) {
-        if (it instanceof Quay quay && quay.getCentroid() != null) {
-          coordinates.add(WgsCoordinateMapper.mapToDomain(quay.getCentroid()));
+      /* try to get a coordinate from quays in this stopPlace */
+      if (stopPlace.getQuays() != null) {
+        for (Object it : stopPlace.getQuays().getQuayRefOrQuay()) {
+          if (it instanceof Quay quay && quay.getCentroid() != null) {
+            coordinates.add(WgsCoordinateMapper.mapToDomain(quay.getCentroid()));
+          }
         }
       }
+      /* FIXME there are "sub"stopPlace with Quays that have no coordinates, but a ParentStopPlaceReference to a parentStop with a coordinate.... */
       if (coordinates.isEmpty()) {
-        throw new IllegalArgumentException(
```

Comment:
So we have a stop place that doesn't have any coordinates and it either doesn't have any quays or they don't have a coordinates either. What sense does it make to include this in the OTP data?

Shouldn't you drop this stop place?

Reply:
This is a work in progress to handle https://github.com/opentripplanner/OpenTripPlanner/issues/4933
There is a coordinate that I have not yet found out how to access from here easily.

Reply:
@sjthiessen @loj-hbt and @bas-hbt seem to be interested in getting this merged. Maybe they can help out with this.

Reply:
I can remove the last two commits and DRY for merging on tuesday. Or someone can cherry-pick [a5229ae](https://github.com/opentripplanner/OpenTripPlanner/pull/4872/commits/a5229ae4a8113bcc5c8fc44a1781508c3bd8a31f) (I think that's what is needed for data around Hamburg)

Reply:
There is a meeting every Thursday as well and we will probably discuss this.